server/commands.go

```diff
@@ -56,74 +56,75 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 					Name:    common.FlagDBPath,
 					Aliases: []string{"f"},
 					Value:   defaultCfg.DatabaseFilePath,
-					Usage:   "File in which to persist Temporal state",
+					Usage:   "File in which to persist Temporal state (by default, Workflows are lost when the process dies)",
 				},
 				&cli.StringSliceFlag{
 					Name:    common.FlagNamespace,
 					Aliases: common.FlagNamespaceAlias,
-					Usage:   "Specify namespaces that should be pre-created. Namespace 'default' is auto created",
+					Usage:   `Specify namespaces that should be pre-created (namespace "default" is always created)`,
 					EnvVars: nil,
 					Value:   nil,
 				},
 				&cli.IntFlag{
 					Name:    common.FlagPort,
 					Aliases: []string{"p"},
-					Usage:   "Port for the temporal-frontend GRPC service",
+					Usage:   "Port for the frontend gRPC service",
 					Value:   sconfig.DefaultFrontendPort,
 				},
 				&cli.IntFlag{
 					Name:  common.FlagMetricsPort,
-					Usage: "Port for the metrics listener",
+					Usage: "Port for /metrics",
 					Value: sconfig.DefaultMetricsPort,
+					DefaultText: "disabled",
 				},
 				&cli.IntFlag{
 					Name:        common.FlagUIPort,
-					Usage:       "port for the temporal web UI",
+					Usage:       "Port for the Web UI",
 					DefaultText: fmt.Sprintf("--port + 1000, eg. %d", sconfig.DefaultFrontendPort+1000),
 				},
 				&cli.BoolFlag{
 					Name:  common.FlagHeadless,
-					Usage: "disable the temporal web UI",
+					Usage: "Disable the Web UI",
 				},
 				&cli.StringFlag{
 					Name:    common.FlagIP,
-					Usage:   `IPv4 address to bind the frontend service to instead of localhost`,
+					Usage:   `IPv4 address to bind the frontend service to`,
 					EnvVars: nil,
 					Value:   "127.0.0.1",
 				},
 				&cli.StringFlag{
 					Name:        common.FlagUIIP,
-					Usage:       `IPv4 address to bind the web UI to instead of localhost`,
-					DefaultText: "same as --ip (eg. 127.0.0.1)",
+					Usage:       `IPv4 address to bind the Web UI to`,
+					DefaultText: "same as --ip",
 				},
 				&cli.StringFlag{
 					Name:    common.FlagLogFormat,
-					Usage:   `customize the log formatting (allowed: ["json" "pretty"])`,
+					Usage:   `Set the log formatting. Options: ["json", "pretty"].`,
 					EnvVars: nil,
 					Value:   "json",
 				},
 				&cli.StringFlag{
 					Name:    common.FlagLogLevel,
-					Usage:   `customize the log level (allowed: ["debug" "info" "warn" "error" "fatal"])`,
+					Usage:   `Set the log level. Options: ["debug" "info" "warn" "error" "fatal"].`,
```

Comment:
no need for the dots at the end

Reply:
periods are to remove double parens and give separation between [] and () and be consistent with the switch to "sentences". 

`customize the log level (allowed: ["debug" "info" "warn" "error" "fatal"]) (default: "info")`

vs

`Set the log level. Options: ["debug" "info" "warn" "error" "fatal"]. (default: "info")`

just my stylistic preference, could remove second period. maybe @djmagee has an opinion

from:

<img width="875" alt="image" src="https://user-images.githubusercontent.com/251288/210696192-d96c60d6-67a4-4ac1-a9c5-bf1a11ff5f88.png">


to:

<img width="978" alt="image" src="https://user-images.githubusercontent.com/251288/210696121-7f765578-dfba-400e-84cc-71d3a3971ef0.png">